addons/purchase_edi_ubl_bis3/models/purchase_edi_xml_ubl_bis3.py

```diff
@@ -178,7 +178,7 @@ def _get_item_vals(self, order, order_line):
 
         vals = {
             'name': product.name,
-            'description': product.description,
+            'description': product.description_purchase,
```

Comment:
Do we pass somewhere order_line.name? Since it contains the manual instruction in the purchase order line, we should really pass it somewhere